src/store/map/index.js

```diff
@@ -133,13 +134,17 @@ const mutations = {
             state.paginateSrpResults.push(state.srpResults[i]);
         }
     },
+
+    'update-filtered-srp-results'(state, srpResults) {
+        state.filteredSrpResults = srpResults;
+    },
 };
 
 const actions = {
     async searchLocation({ commit }, query) {
         const token =
             'pk.eyJ1IjoiaWFtZmlhc2NvIiwiYSI6ImNrOWZiankzdjA5d2kzbWp3NGNzNmIwaHAifQ.E2UwYdvpjc6yNoCmBjfTaQ';
-        const url = `/geocoding/v5/mapbox.places/${query}.json?access_token=${token}&proximity=77.4977,12.9716`;
+        const url = `/geocoding/v5/mapbox.places/${query}.json?access_token=${token}&worldview=in&country=in`;
```

Comment:
tested this?

Reply:
proximity restricting searching in other places...now restricted to india...if typed correctly will show correct result

Reply:
What do you mean by "typed correctly"
Most of our searches is around bangalore (thanks to google ads configuration)
So will the fuzzy search quality improve or drop?